### ISSUE TITLE
Tabs and Pagination don't work well together

### ISSUE THREAD

**Issue by DouglasWebster** [valor-software/ngx-bootstrap#1644]
The Tabs and Pagination modules do not work well when included in the same component. 

The Dynamic tabs example is shown as using changeDetection: ChangeDetectionStrategy.OnPush but this interferes with the synchronisation of the pager functionality; it is always on click behind! This can be seen in this [plunker](https://embed.plnkr.co/Uk4NTv/).  Just alternative click the two pager buttons!

I raised this before but then I probably got a bit wordy and didn't know how to make a simple plunker to show the problem so it might not have been clear.  Hopefully this short demo will clarify things.

Not sure how often a Pager and a Dynamic Tab will be used together but I thought it best that you were aware of the problem.

I think this project is awesome - thank you.

**Comment by valorkin** [comment]
@DouglasWebster thanks you for kind words!
tabs impl is an oldy sh*t at the moment
I am planning to rewrite them (just after new datepicker with popup and typeahead multiselect)
I have already tried it, and it works much better with dynamic components

**Comment by IlyaSurmay** [comment]
Fixed in 2.0.0-beta.8
